holdings_maintenance/pds4/pds4infoshelf.py

```
@@ -0,0 +1,931 @@
+
+def reinitialize(pdsdir, selection=None, logger=None):
+
+    info_path = pdsdir.shelf_path_and_lskip('info')[0]
+
+    # Warn if shelf file does not exist
+    if not os.path.exists(info_path):
+        logger = logger or pdslogger.PdsLogger.get_logger(LOGNAME)
+        if selection:
+            logger.error('Info shelf file does not exist', info_path)
+        else:
+            logger.warn('Info shelf file does not exist; initializing',
```

Ruff (0.14.0)

Logging statement uses `warn` instead of `warning`

Convert to `warning`

(G010)